src/plone/restapi/services/types/configure.zcml

```diff
@@ -11,6 +11,39 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="PATCH"
+    for="*"
+    factory=".update.TypesUpdate"
+    name="@types"
+    permission="plone.app.controlpanel.Overview"
```

Comment:
Have you checked what permission we use in Plone for the dexterity types control panel? Is that p.a.controlpanel.Overview?

Reply:
In Plone it is:
`permission="plone.schemaeditor.ManageSchemata"` which is by default assigned to `Manager`

But as in `plone.restapi` we didn't introduce hard-dependency on `plone.schemaeditor` we used the same permission we use for all the other control-panels `plone.app.controlpanel.Overview` (default `Manager`, `Site Administrator`)